pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+PHEP authors are responsible for collecting community feedback on a PHEP before submitting it for review. However, to avoid long-winded and open-ended discussions, strategies such as soliciting private or more narrowly-tailored feedback in the early design phase, collaborating with other community members with expertise in the PHEP's subject matter, and picking an appropriately-specialized discussion for the PHEP's topic (if applicable) should be considered. PHEP authors should use their discretion here.
+
+PHEP discussions are subject to the [PyHC Code of Conduct](https://heliopython.org/docs/code_of_conduct/).
+
+## PHEP Review & Resolution
+Once the authors have completed a PHEP, they may request a review for style and consistency from the PHEP editors. However, content review and acceptance of the PHEP is ultimately the responsibility of the community once the authors determine the PHEP is ready for final review and resolution. A PHEP is accepted when approved by consensus of attendees (virtual or in-person) at both a PyHC regular telecon and a regular PyHC meeting with no edits to the PHEP occurring in between. Any edits "reset the clock" and two further votes must be taken. Potential approval of a PHEP must be placed on the agenda at least a month before the meeting, including a link to the open PR with the proposed-final language of the PHEP. Persons not in attendance at the meetings may object via email or Element/Slack chat, provided that they ensure the PHEP author is notified.
+
+A "regular PyHC meeting" is any meeting (virtual, physical, or hybrid) organized for the purpose of discussing PyHC business, of at least a half-day duration, where a substantial portion of the community is expected to participate, and announced as such well in advance. This includes the semiannual meetings but may also include side gatherings at other meetings which are of interest to the community and designated as PyHC meetings.
+
+"Consensus" here means no concrete objections have been raised: an accepted PHEP must be one that the entire community can at least live with. The PHEP authors must make a compelling case for the PHEP; objectors must provide meaningful concerns that could, in principle, be addressed by the authors. All are expected to discuss in good faith, while recognizing that good faith does not guarantee consensus can be reached.
+
+For a PHEP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate PyHC projects unduly. Finally, a proposed enhancement must be "pythonic" in order to be accepted by the community. (However, "pythonic" is an imprecise term; it may be defined as whatever is acceptable to the community. This logic is intentionally circular.)
+
+The editors must post oronouncements of PHEP resolution to the PyHC mailing list.
```

Comment:
I've already caught this typo and it will be fixed in a future push.

Reply:
Or perhaps we could adopt "oronouncements" as the official declaration that a PHEP has been accepted?  😸